pom.xml

```diff
@@ -28,19 +28,32 @@
 		<dependency>
 			<groupId>org.projectlombok</groupId>
 			<artifactId>lombok</artifactId>
+			<optional>true</optional>
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Why spring-boot-starter-data-jpa ?

Reply:
This dependency was allowing me to use javax.persistence package and so I added it